### PR TITLE
Add nix snippets

### PR DESCRIPTION
**Pull request by doronbehar**

### CODE REVIEW

File: snippets/nix.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        "body": [
+            "let",
+            "  $1",
+            "in $0"
+        ],
+        "description": "let-in block"
+    },
+    "hash": {
+        "prefix": "hash",
+        "body": "hash = \"sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=\";",
```
Comment (OkelleyDevelopment) [inline snippets/nix.json:22]: Should the body be `hash = ${1:sha256-AAA...}`?

Comment (doronbehar) [reply snippets/nix.json]: > Should the body be `hash = ${1:sha256-AAA...}`?

Yea why not :). Thanks.

Comment (doronbehar) [reply snippets/nix.json]: Pushed this change.

**Review by OkelleyDevelopment — APPROVED** [review]
Thanks for the quick edit!

**Pull request merged**